ml4gw/distributions.py

```diff
@@ -73,6 +73,20 @@ def __call__(self, N: int) -> torch.Tensor:
         return x
 
 
+class LogUniform(Uniform):
+    """
+    Sample from a log uniform distribution
+    """
+
+    def __init__(self, low: float, high: float) -> None:
+        self.low = math.log(low)
+        self.high = math.log(high)
+
+    def __call__(self, N: int) -> torch.Tensor:
+        u = super().__call__(N)
```

Comment:
If you're gonna take advantage of this as a child class, might be worth considering using `super.__init__(math.log(low), math.log(high))` above to ensure total consistency